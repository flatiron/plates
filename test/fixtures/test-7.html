<a id="someid1" class="name" href="/foo"></a>
<a id="someid2" onclick="name" href="/foo"></a>