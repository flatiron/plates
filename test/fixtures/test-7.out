<a id="someid1" class="name" href="/foo">Alpha</a>
<a id="someid2" onclick="name" href="/foo"></a>